### PR TITLE
Speech_Recognition

### PR DESCRIPTION
**Pull request by Tudor-Gavriliuc**

### CODE REVIEW

File: Speech_Recognition/src/Requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Python version
+The "SpeechRecognition" library, which is commonly used for speech recognition in Python, is compatible with Python 3.x versions. It's recommended to use Python 3.3 or later when working with this library.
+
+# Speech recognition libraries
+pyaudio==0.2.11        # For audio input/output
+SpeechRecognition==3.8.1  # For speech recognition functionality
```
Comment (pyramixofficial) [inline Speech_Recognition/src/Requirements.txt:6]: Please adapt your file.
the naming should be with only lowercase letter `requirements.txt`
Make sure to only include the libraries and it's version, comments around what are they used for can be added in the code part when you import them if you feel like it

File: Speech_Recognition/src/img/0_Mqx9bNxzLtNWWm1B.jpg
Comment (pyramixofficial) [inline Speech_Recognition/src/img/0_Mqx9bNxzLtNWWm1B.jpg:1]: Please rename this and all the other images with relevant namings around it that explain what can I find in that file specifically

File: Speech_Recognition/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    ```
+
+## Source Code Explanation
+
+
+- `main.ipynb`: This file contains the main code for your Speech Recognitiion application. 
+
+- `img/`: This folder contains images used the article.
+
+- `links`: This folder contains links used the article.
```
Comment (pyramixofficial) [inline Speech_Recognition/README.md:53]: Please updated this line, currently links is a undefined file which has in it all the links, my suggestion would be to add all the links in the README in a separate section and to refference them in the article side

File: Speech_Recognition/medium_speech_recognition.pdf
Comment (pyramixofficial) [inline Speech_Recognition/medium_speech_recognition.pdf:1]: No need to add the pdf version if you have the article.md formatted accordingly in the same way

File: Speech_Recognition/article.md
Comment (pyramixofficial) [inline Speech_Recognition/article.md:1]: Please write the article in the markdown version inside this file(the entire content)

**Review by pyramixofficial — CHANGES_REQUESTED** [review]
Just added more comments around your work, please adapt the content

File: Speech_Recognition/README.md (hunk 2 above)
Comment (Tudor-Gavriliuc) [reply Speech_Recognition/README.md]: Should I add the links used in my medium article?

**Review by tokyo-s — APPROVED** [review]
Good job!

**Review by tokyo-s — DISMISSED** [review]
All cool!
could you remove all the additional text from requirements.txt? 
usually in this file there are only rows with libs and versions, nothing more, not even comments
about python version I see you wrote in README.md, so this is enough, thanks!

**Review by tokyo-s — COMMENTED** [review]
Also, I see you used word perevodcik, but in code there is actually written peverodcik or smthing like that, I recommend to put the name 'translator' or something which is more explainable

File: Speech_Recognition/README.md (hunk 2 above)
Comment (pyramixofficial) [reply Speech_Recognition/README.md]: not necessarily, just make sure that the images will be displayed on this page and that on medium the coresponding images will be placed there

**Comment by pyramixofficial** [comment]
The videos are really great but let's assume that I'm a reader in the bus and I can't get my headphones out for the moment, but I would still like to hear and enjoy the content, I would suggest to add the relevant information from the video in the article as well maybe in a sectioned thing so that the user could only stick to the text in case of need.

**Review by pyramixofficial — APPROVED** [review]

**Review by pyramixofficial — APPROVED** [review]

**Pull request merged**